pkg/athenz/model_test.go

```diff
@@ -16,6 +16,75 @@ func toRDLTimestamp(s string) (rdl.Timestamp, error) {
 	return rdl.TimestampParse(s)
 }
 
+func TestGetRolesForDomain(t *testing.T) {
+
+	modified, err := toRDLTimestamp("2018-03-14T19:36:41.003Z")
+	if err != nil {
+		t.Errorf(err.Error())
+	}
+	cases := []struct {
+		test     string
+		domain   *zms.DomainData
+		expected Roles
+	}{
+		{
+			test:     "nil athenz domain",
+			domain:   nil,
+			expected: make(Roles, 0),
+		},
+		{
+			test:     "empty domain roles",
+			domain:   &zms.DomainData{},
+			expected: make(Roles, 0),
+		},
+		{
+			test: "valid Athenz domain",
+			domain: &zms.DomainData{
+				Name: "athenz-domain.name",
+				Roles: []*zms.Role{
+					{
+						Name:     "athenz-domain.name:role.my-reader-role",
+						Modified: &modified,
+						RoleMembers: []*zms.RoleMember{
+							{
+								MemberName: "client-domain.serviceA",
+							},
+							{
+								MemberName: "client-domain.serviceB",
+							},
+							{
+								MemberName: "client2-domain.serviceA",
+							},
+						},
+					},
+					{
+						Name:     "athenz-domain.name:role.my-writer-role",
+						Modified: &modified,
+						RoleMembers: []*zms.RoleMember{
```

Comment:
Should we get rid of the Modified and Role members structs here as they're not actually being used in the unit test?